charts/csi-cloudscale/templates/daemonset.yaml

```diff
@@ -0,0 +1,107 @@
+---
+kind: DaemonSet
+apiVersion: apps/v1
+metadata:
+  name: {{ include "csi-cloudscale.driver-name" . }}-node
+spec:
+  selector:
+    matchLabels:
+      app: csi-cloudscale-node
+  template:
+    metadata:
+      labels:
+        app: csi-cloudscale-node
+        role: csi-cloudscale
+    spec:
+      priorityClassName: system-node-critical
+      serviceAccount: {{ include "csi-cloudscale.node-service-account-name" . }}
+      hostNetwork: true
+      containers:
+        - name: csi-node-driver-registrar
+          image: {{ .Values.driverRegistrar.image.repository }}:{{ .Values.driverRegistrar.image.tag }}
+          imagePullPolicy: {{ .Values.driverRegistrar.image.pullPolicy }}
+          args:
+            - "--v=5"
+            - "--csi-address=$(ADDRESS)"
+            - "--kubelet-registration-path=$(DRIVER_REG_SOCK_PATH)"
+          lifecycle:
+            preStop:
+              exec:
+                command: ["/bin/sh", "-c", "rm -rf /registration/csi.cloudscale.ch /registration/csi.cloudscale.ch-reg.sock"]
+          env:
+            - name: ADDRESS
+              value: /csi/csi.sock
+            - name: DRIVER_REG_SOCK_PATH
+              value: /var/lib/kubelet/plugins/csi.cloudscale.ch/csi.sock
+            - name: KUBE_NODE_NAME
+              valueFrom:
+                fieldRef:
+                  fieldPath: spec.nodeName
+          {{- with .Values.driverRegistrar.resources }}
+          resources:
+{{ toYaml . | indent 12 }}
```

Comment:
Hahaha, this looks like something straight out of hell :) Something @feuermurmel would probably enjoy. :)